stats.cpp

```diff
@@ -1,5 +1,28 @@
 #include "stats.h"
+class Stats
+{float average;
+ float max;
+ float min;
+};
 
-Stats Statistics::ComputeStatistics(const std::vector<___>& ) {
+Stats Statistics::ComputeStatistics(const std::vector<float>& a) {
     //Implement statistics here
+    Stats s;
+    float sum=0,maxi=0,mini=99999;
```

Comment:
min and max will not work for certain combinations of numbers. Try to make a test case to unearth the fault :)